src/whos_there/senders/slack.py

```diff
@@ -6,25 +6,37 @@
 
 
 class SlackSender(Sender):
-    def __init__(self, webhook_url: str, channel: str, user_mentions: list[str] | None = None) -> None:
+    def __init__(
+        self,
+        webhook_url: str,
+        channel: str,
+        user_mentions: list[str] | None = None,
+        sender_username: str = "Who's There notification",
+        icon_emoji: str = ":bell:",
+    ) -> None:
         """Initialize the Slack sender.
 
         Args:
-            webhook_url: The Slack webhook URL.
-            channel: The Slack channel name.
-            user_mentions: The list of users to mention.
+            webhook_url (str): The Slack webhook URL.
+            channel (str): The Slack channel name.
+            sender_username (str): User name used for notifications (defaults to "Who's There notification").
+            user_mentions (List[str]): The list of users to mention (defaults to None).
+            icon_emoji (str): Emoji to use for the notifications (defaults to ":bell:")
```

Comment:
```suggestion
            webhook_url: The Slack webhook URL.
            channel: The Slack channel name.
            sender_username: User name used for notifications.
            user_mentions: The list of users to mention.
            icon_emoji: Emoji to use for the notifications.
```
I prefer the notypes option